src/cli.cpp

```diff
@@ -218,16 +222,26 @@ void CLI::printID(DWORD *pid)
     fmt::print("Product ID: PPPPP-{}-{}-23xxx\n", b, c);
 }
 
-void CLI::printKey(char *pk) {
+void CLI::printKey(char *pk, Options options) {
     assert(strlen(pk) == 25);
 
     std::string spk = pk;
-    fmt::print("{}-{}-{}-{}-{}\n",
-               spk.substr(0,5),
-               spk.substr(5,5),
-               spk.substr(10,5),
-               spk.substr(15,5),
-               spk.substr(20,5));
+    if (options.unformatted == true) {
+        fmt::print("{}-{}-{}-{}-{}",
+            spk.substr(0, 5),
+            spk.substr(5, 5),
+            spk.substr(10, 5),
+            spk.substr(15, 5),
+            spk.substr(20, 5));
+    }
+    else {
+        fmt::print("{}-{}-{}-{}-{}\n",
+            spk.substr(0, 5),
+            spk.substr(5, 5),
+            spk.substr(10, 5),
+            spk.substr(15, 5),
+            spk.substr(20, 5));
+    }
```

Comment:
Clean up this if statement
```suggestion
	fmt::print("{}-{}-{}-{}-{}",
		spk.substr(0, 5),
		spk.substr(5, 5),
		spk.substr(10, 5),
		spk.substr(15, 5),
		spk.substr(20, 5));
	
	if (!options.unformatted) {
		fmt::print("\n");
	}
```